### ISSUE TITLE
Release version 1.0.0

### ISSUE THREAD

**Issue by pcchen** [Cytnx-dev/Cytnx#542]
Since we would like to release version 1.0.0 soon, please help to list the issues which must be fixed before we can release it. Please also mark the process, such as on-going, in PR, merged into dev-master, etc.

**Comment by IvanaGyro** [comment]
#534 fixed the build process while enabling CUDA. It must be merged before release. Tag @hunghaoti for notification.

**Comment by IvanaGyro** [comment]
https://github.com/Cytnx-dev/Cytnx/pull/528/commits/562f093c2d00b4bf39f1e5643bca973fb1174613 marks a *public (but may not be an API)* function as deprecated. We may also want to cherry-pick the commit.

**Comment by hunghaoti** [comment]
> #534 fixed the build process while enabling CUDA. It must be merged before release. Tag @hunghaoti for notification.

I have merged it.

**Comment by ianmccul** [comment]
[[[deprecated]]](https://en.cppreference.com/w/cpp/language/attributes/deprecated) attribute

**Comment by IvanaGyro** [comment]
Please ignore the comments I left above. Per the offline discussion, we are putting our C++ API into unstable mode. We will change the C++ API without notification, which means changing the C++ API will not need to pump the version up and will not block this release.

**Comment by manuschneider** [comment]
We made some changes in the API, which should also be changed in the API documentation. These are:
* Contract, Contracts are merged into Contract
* Contract_, Contracts_ are merged into Contract_
* .relabel, .relabels are merged into .relabel
* .relabel_, .relabels_ are merged into .relabel_
* Svd_truncate has an additional optional argument min_blockdim to define a minimum dimension for each block

The code examples need to be changed accordingly such that all code works with version 1.0

**Comment by hunghaoti** [comment]
I will write in the [`release note`](https://github.com/Cytnx-dev/Cytnx/blob/master/misc_doc/version.log).

**Comment by hunghaoti** [comment]
Thank you for all your comments. I have addressed them and released version [v1.0.0](https://github.com/Cytnx-dev/Cytnx/releases/tag/v1.0.0)
